### PR TITLE
Jenkins Awards 2023 - adding Jenkins award nominees

### PR DESCRIPTION
**Pull request by kmartens27**
This PR is to add the list of jenkins award nominees to their respective category, and provide a link directly to the google form for voting

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -21,17 +21,20 @@ https://github.com/jenkinsci/
...
-- Jenkins Security MVP → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6034)
+Candidatess are:
+- Most Valuable Jenkins Advocate → [Kris Stern](https://github.com/krisstern), [Alexander Brandes](https://github.com/NotMyFault), [Bruno Verachten](https://github.com/gounthar), [Alex Earl](https://github.com/slide), [Jean-Marc Meessen](https://github.com/jmMeessen), and [Mark Waite](https://github.com/MarkEWaite)
+[Voting is open](https://github.com/jenkins-infra/jenkins.io/issues/6035)
+- Most Valuable Jenkins Contributor → [Damien Duportal](https://github.com/dduportal), [Alexander Brandes](https://github.com/NotMyFault), and [Jan Faracik](https://github.com/janfaracik)
+[Voting is open](https://github.com/jenkins-infra/jenkins.io/issues/6033)
+- Jenkins Security MVP → [Damien Duportal](https://github.com/dduportal), [Devin Nusbaum](https://github.com/dwnusbaum), [Daniel Beck](https://github.com/daniel-beck), [Valdes Che Zogou](https://github.com/ValdesChe), and [Kevin Guerroudj](https://github.com/Kevin-CB)
+[Voting is open](https://github.com/jenkins-infra/jenkins.io/issues/6034)
```
Comment (lemeurherve) [inline profile/README.md:32]:
```suggestion
- Most Valuable Jenkins Advocate → [Alex Earl](https://github.com/slide), [Alexander Brandes](https://github.com/NotMyFault), [Bruno Verachten](https://github.com/gounthar), [Jean-Marc Meessen](https://github.com/jmMeessen), [Kris Stern](https://github.com/krisstern), and [Mark Waite](https://github.com/MarkEWaite)
[Voting is open](https://github.com/jenkins-infra/jenkins.io/issues/6035)
- Most Valuable Jenkins Contributor → [Alexander Brandes](https://github.com/NotMyFault), [Damien Duportal](https://github.com/dduportal), and [Jan Faracik](https://github.com/janfaracik)
[Voting is open](https://github.com/jenkins-infra/jenkins.io/issues/6033)
- Jenkins Security MVP → [Damien Duportal](https://github.com/dduportal), [Daniel Beck](https://github.com/daniel-beck), [Devin Nusbaum](https://github.com/dwnusbaum), [Kevin Guerroudj](https://github.com/Kevin-CB), and [Valdes Che Zogou](https://github.com/ValdesChe)
[Voting is open](https://github.com/jenkins-infra/jenkins.io/issues/6034)
```
Proposing to sort names alphabetically

File: profile/README.md [hunk 2]
```diff
@@ -21,17 +21,20 @@ https://github.com/jenkinsci/
 
 ### Jenkins Contributor Awards 2023
 
-The annual Jenkins awards is happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
+The annual Jenkins awards are happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins. You can vote by filling out the [Google form](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform).
```
Comment (lemeurherve) [inline profile/README.md:24]:
```suggestion
The annual Jenkins awards are happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
Vote now by filling out the [Google form](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform).
```
One line per phrase for easier review, and more active formulation.

File: profile/README.md [hunk 3]
```diff
@@ -21,17 +21,20 @@ https://github.com/jenkinsci/
...
 
-Nomination categories are:
-- Most Valuable Jenkins Advocate → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6035)
-- Most Valuable Jenkins Contributor → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6033)
-- Jenkins Security MVP → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6034)
+Candidatess are:
```
Comment (lemeurherve) [inline profile/README.md:26]:
```suggestion
Nominated candidatess are:
```

**Comment by kmartens27** [comment]
That makes a ton of sense @lemeurherve, thanks for catching that. I've implemented the alphabetical listing. I had gone off the way the nominees are displayed in the google form, but this is much nicer.

File: profile/README.md [hunk 4]
```diff
@@ -21,17 +21,20 @@ https://github.com/jenkinsci/
...
+- Most Valuable Jenkins Advocate → [Alex Earl](https://github.com/slide), [Alexander Brandes](https://github.com/NotMyFault), [Bruno Verachten](https://github.com/gounthar), [Jean-Marc Meessen](https://github.com/jmMeessen), [Kris Stern](https://github.com/krisstern), and [Mark Waite](https://github.com/MarkEWaite)
+[Voting is open](https://github.com/jenkins-infra/jenkins.io/issues/6035)
+- Most Valuable Jenkins Contributor → [Alexander Brandes](https://github.com/NotMyFault), [Damien Duportal](https://github.com/dduportal), and [Jan Faracik](https://github.com/janfaracik)
+[Voting is open](https://github.com/jenkins-infra/jenkins.io/issues/6033)
+- Jenkins Security MVP → [Damien Duportal](https://github.com/dduportal), [Daniel Beck](https://github.com/daniel-beck), [Devin Nusbaum](https://github.com/dwnusbaum), [Kevin Guerroudj](https://github.com/Kevin-CB), and [Valdes Che Zogou](https://github.com/ValdesChe)
+[Voting is open](https://github.com/jenkins-infra/jenkins.io/issues/6034)
```
Comment (lemeurherve) [inline profile/README.md:32]:
```suggestion
- [Most Valuable Jenkins Advocate](https://github.com/jenkins-infra/jenkins.io/issues/6035) → [Alex Earl](https://github.com/slide), [Alexander Brandes](https://github.com/NotMyFault), [Bruno Verachten](https://github.com/gounthar), [Jean-Marc Meessen](https://github.com/jmMeessen), [Kris Stern](https://github.com/krisstern), and [Mark Waite](https://github.com/MarkEWaite)
[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
- [Most Valuable Jenkins Contributor ](https://github.com/jenkins-infra/jenkins.io/issues/6033)→ [Alexander Brandes](https://github.com/NotMyFault), [Damien Duportal](https://github.com/dduportal), and [Jan Faracik](https://github.com/janfaracik)
[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
- [Jenkins Security MVP](https://github.com/jenkins-infra/jenkins.io/issues/6034) → [Damien Duportal](https://github.com/dduportal), [Daniel Beck](https://github.com/daniel-beck), [Devin Nusbaum](https://github.com/dwnusbaum), [Kevin Guerroudj](https://github.com/Kevin-CB), and [Valdes Che Zogou](https://github.com/ValdesChe)
[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
```
More link to the voting form

**Comment by kmartens27** [comment]
Awesome, thanks so much @lemeurherve, I've implemented all the changes suggested, and added the ! to the first section to set it off more. Let me know if there's anything else to change/update, otherwise it should be all set!

File: profile/README.md [hunk 5]
```diff
@@ -21,17 +21,21 @@ https://github.com/jenkinsci/
...
+[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
+- [Most Valuable Jenkins Contributor ](https://github.com/jenkins-infra/jenkins.io/issues/6033)→ [Alexander Brandes](https://github.com/NotMyFault), [Damien Duportal](https://github.com/dduportal), and [Jan Faracik](https://github.com/janfaracik)
+[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
+- [Jenkins Security MVP](https://github.com/jenkins-infra/jenkins.io/issues/6034) → [Damien Duportal](https://github.com/dduportal), [Daniel Beck](https://github.com/daniel-beck), [Devin Nusbaum](https://github.com/dwnusbaum), [Kevin Guerroudj](https://github.com/Kevin-CB), and [Valdes Che Zogou](https://github.com/ValdesChe)
+[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
```
Comment (lemeurherve) [inline profile/README.md:33]:
```suggestion
- [Most Valuable Jenkins Advocate](https://github.com/jenkins-infra/jenkins.io/issues/6035) → [Alex Earl](https://github.com/slide), [Alexander Brandes](https://github.com/NotMyFault), [Bruno Verachten](https://github.com/gounthar), [Jean-Marc Meessen](https://github.com/jmMeessen), [Kris Stern](https://github.com/krisstern) and [Mark Waite](https://github.com/MarkEWaite): 
[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
- [Most Valuable Jenkins Contributor ](https://github.com/jenkins-infra/jenkins.io/issues/6033)→ [Alexander Brandes](https://github.com/NotMyFault), [Damien Duportal](https://github.com/dduportal) and [Jan Faracik](https://github.com/janfaracik): 
[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
- [Jenkins Security MVP](https://github.com/jenkins-infra/jenkins.io/issues/6034) → [Damien Duportal](https://github.com/dduportal), [Daniel Beck](https://github.com/daniel-beck), [Devin Nusbaum](https://github.com/dwnusbaum), [Kevin Guerroudj](https://github.com/Kevin-CB) and [Valdes Che Zogou](https://github.com/ValdesChe): 
[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
```
Another suggestion, adding `: ` as the voting link are right after the last nominated and not on their own line

**Review by lemeurherve — APPROVED** [review]
👍

File: profile/README.md (hunk 5 above)
Comment (kmartens27) [reply profile/README.md]: That makes sense, I've added that to my edits!

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**